packages/cli-e2e/__tests__/atomic.specs.ts

```diff
@@ -144,14 +144,12 @@ describe('ui:create:atomic', () => {
     }, 60e3);
 
     it('should retrieve the search token on the page load', async () => {
-      const tokenResponseListener = page.waitForResponse(tokenServerEndpoint);
-
-      page.goto(searchPageEndpoint);
-      await page.waitForSelector(searchInterfaceSelector);
+      const response = await page.goto(tokenServerEndpoint, {
```

Comment:
test now passes but is a bit flaky...

@louis-bompart Is the test setup I have (before, after) optimal?

Reply:
Heu, that test does not do what it says it does.
with a page.goto you're navigating to the token server in Chrome, you're not testing that you indeed get a search token when you load the page from the create atomic.
So... Ain't even a question of 'optimal' here, it's just not testing what it should.